diff --git a/node_modules/@univerjs/engine-formula/lib/es/index.js b/node_modules/@univerjs/engine-formula/lib/es/index.js
index 18d971c..24c3617 100644
--- a/node_modules/@univerjs/engine-formula/lib/es/index.js
+++ b/node_modules/@univerjs/engine-formula/lib/es/index.js
@@ -1,5 +1,5 @@
 import { AbsoluteRefType, AsyncLock, BooleanNumber, BuildTextUtils, CellValueType, CommandType, DataStreamTreeTokenType, Disposable, DisposableCollection, ICommandService, IConfigService, IUniverInstanceService, Inject, Injector, LRUMap, LifecycleService, LocaleService, LocaleType, MAX_COLUMN_COUNT, MAX_ROW_COUNT, ObjectMatrix, Optional, Plugin, RANGE_TYPE, RTree, Rectangle, RichTextBuilder, Tools, UniverInstanceType, cellToRange, columnLabelToNumber, createIdentifier, generateRandomId, getNumfmtParseValueFilter, hashAlgorithm, isFormulaId, isFormulaString, isRealNum, isTextFormat, isValidRange, merge, moveRangeByOffset, numfmt, requestImmediateMacroTask, sortRules, toDisposable, touchDependencies } from "@univerjs/core";
-import { BehaviorSubject, Observable, Subject, bufferWhen, combineLatest, distinctUntilChanged, filter, map, shareReplay } from "rxjs";
+import { BehaviorSubject, Observable, Subject, bufferWhen, combineLatest, distinctUntilChanged, filter, map, shareReplay, skip } from "rxjs";
 import IntervalTree from "@flatten-js/interval-tree";
 import Decimal from "decimal.js";
 import { DataSyncPrimaryController } from "@univerjs/rpc";
@@ -40788,7 +40788,7 @@ let RegisterOtherFormulaService = class RegisterOtherFormulaService extends Disp
 				this._commandService.executeCommand(OtherFormulaMarkDirty.id, { [unitId]: { [subUnitId]: { [formulaId]: true } } }, { onlyLocal: true });
 			});
 		};
-		this.disposeWithMe(this._formulaChangeWithRange$.pipe(bufferWhen(() => this.calculateStarted$.pipe(filter((calculateStarted) => calculateStarted)))).subscribe((options) => options.forEach(handleRegister)));
+		this.disposeWithMe(this._formulaChangeWithRange$.pipe(bufferWhen(() => this.calculateStarted$.pipe(skip(1), filter((calculateStarted) => calculateStarted)))).subscribe((options) => options.forEach(handleRegister)));
 		this.disposeWithMe(this._formulaChangeWithRange$.pipe(filter(() => this.calculateStarted$.getValue())).subscribe(handleRegister));
 	}
 	_initFormulaCalculationResultChange() {
