diff --git a/node_modules/@univerjs/sheets-ui/lib/es/index.js b/node_modules/@univerjs/sheets-ui/lib/es/index.js
index 0000000..1111111 100644
--- a/node_modules/@univerjs/sheets-ui/lib/es/index.js
+++ b/node_modules/@univerjs/sheets-ui/lib/es/index.js
@@ -10584,7 +10584,7 @@
 		return this._submitEdit(documentDataModel.getSnapshot());
 	}
 	async _submitEdit(snapshot, wholeSelection = false) {
-		var _this$_workbookSelect;
+		var _worksheet$getCellRaw, _this$_workbookSelect;
 		const editCellState = this._editorBridgeService.getEditCellState();
 		if (editCellState == null) return true;
 		const { unitId, sheetId, row, column } = editCellState;
@@ -10595,8 +10595,9 @@
 		worksheet = workbook.getActiveSheet();
 		const cellData = getCellDataByInput({ ...worksheet.getCellRaw(row, column) || {} }, snapshot, this._lexerTreeBuilder, this._localService, this._functionService, workbook.getStyles());
 		if (!cellData) return true;
+		const rawEditorCellData = getCellDataByInput({ s: (_worksheet$getCellRaw = worksheet.getCellRaw(row, column)) === null || _worksheet$getCellRaw === void 0 ? void 0 : _worksheet$getCellRaw.s }, snapshot, this._lexerTreeBuilder, this._localService, this._functionService, workbook.getStyles());
 		this._removeComposedCellStyleInCellData(cellData, worksheet.getComposedCellStyleWithoutSelf(row, column));
-		const finalCell = this._sheetInterceptorService.onWriteCell(workbook, worksheet, row, column, cellData);
+		const finalCell = this._sheetInterceptorService.onWriteCell(workbook, worksheet, row, column, cellData, rawEditorCellData);
 		const finalCellCleaned = cleanCellDataObject(finalCell);
 		if (finalCellCleaned === null || finalCellCleaned === void 0 ? void 0 : finalCellCleaned.s) finalCellCleaned.s = workbook.getStyles().get(finalCellCleaned.s);
 		const rawCellCleaned = cleanCellDataObject(worksheet.getCellRaw(row, column));
